pentaho5/src/main/java/pt/webdetails/cpk/InterPluginBroker.java

```diff
@@ -45,11 +48,18 @@ public static void run( Map<String, Object> params, OutputStream out ) throws Ex
     }
 
     IPluginCall pluginCall = getPluginCall();
+    if ( pluginCall == null ) {
+      final String pluginId = CorePlugin.CDE.getId();
+      final String beanId = getSettingValue( CDE_RENDER_API_BEAN_ID_TAG, CDE_RENDER_API_BEAN_ID );
+      final String method = getSettingValue( CDE_RENDER_API_RENDER_METHOD_TAG, CDE_RENDER_API_RENDER_METHOD );
 
-    String response = pluginCall != null
-      ? pluginCall.call( parameters.getParameters() )
-      : "No valid InterPluginCall available";
+      logger.error( "No valid InterPluginCall available" +
```

Comment:
The log should be on each try of finding the bean or we shouldn't include information about the beanId and method on the log. I'll explain the reasoning better tomorrow in person.